game/building/src/widgets/BuildPanel/widgets/DropLight/components/ColorSelect/index.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
```

Comment:
not sure how I feel about unfinished code being checked in.  Makes it hard to review properly, for instance I can't know if the use of let in renderColorBar is valid, but if allowed to be checked in, next PR won't show those as being changes so I won't know to review them.